### PR TITLE
 [Task] Add zip creation based on folder ids

### LINKED ISSUE #473: [Task] Add endpoint to ZIP-Download to download whole folder
See #440

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request
Resolves #473 

## Additional info
Adding export endpoint to export zip based on folder ids

### CODE REVIEW

File: src/Asset/Controller/Export/ZipAssetController.php [hunk 1]
```diff
@@ -55,31 +55,31 @@ public function __construct(
...
+        content: new ScalarItemsJson('integer', 'assets')
     )]
     #[CreatedResponse(
-        description: 'asset_create_zip_created_response',
+        description: 'asset_export_zip_created_response',
         content: new IdJson('ID of created jobRun', 'jobRunId')
     )]
     #[DefaultResponses([
         HttpResponseCodes::UNAUTHORIZED,
         HttpResponseCodes::NOT_FOUND,
```
Comment (martineiber) [inline src/Asset/Controller/Export/ZipAssetController.php:76]: Missing HttpResponseCodes::MAX_FILE_SIZE_EXCEEDED? b/c of MaxFileSizeExceededException

File: src/Asset/Controller/Export/ZipFolderController.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        tags: [Tags::Assets->name]
+    )]
+    #[ZipExportFolderRequestBody]
+    #[CreatedResponse(
+        description: 'asset_export_zip_created_response',
+        content: new IdJson('ID of created jobRun', 'jobRunId')
+    )]
+    #[DefaultResponses([
+        HttpResponseCodes::UNAUTHORIZED,
+        HttpResponseCodes::NOT_FOUND,
```
Comment (martineiber) [inline src/Asset/Controller/Export/ZipFolderController.php:73]: Also missing HttpResponseCodes::MAX_FILE_SIZE_EXCEEDED ?

**Review by martineiber — CHANGES_REQUESTED** [review]

File: src/Asset/Controller/Export/ZipAssetController.php (hunk 1 above)
Comment (mattamon) [reply src/Asset/Controller/Export/ZipAssetController.php:76]: Good catch! I will add it

**Review by martineiber — APPROVED** [review]
LGTM :duck:

**Review by lukmzig — APPROVED** [review]

**Pull request merged**